README.md

```diff
@@ -1 +1,46 @@
-# hw01
\ No newline at end of file
+# hw01
+title: "Hw 1"
+author: Zoe Cullison-Shimada  
+  
+# hello my name is?
+I was born and raised in **New York**.
+I am a rising **third-year**.
+I am interested in **languages**. I know _French_ and _Spanish_ but I would love to learn _Japanese_.  
+  
+# something I'd like to learn about? 
+I'd like to take a philsophy class because I've never taken one before. 
+I'd like to investigate my thoughts and carry out a creative inquiry about the world. Thus, a project of mine  
+  
+*The cultural and philsophical undertones of introductions*  
+
+* forms of introduction 
+  + verbal greetings
+  + visual greetings or eye-contact
+  + hug or handshake  
+A footnote [^1]  
```

Comment:
Footnotes are hard to implement directly in Markdown. You can use them in R Markdown documents and R will basically rewrite the code to make the footnotes actually appear at the bottom of the document. Nice attempt though!